### ISSUE TITLE
Homepage 

### ISSUE THREAD

**Issue by rgnarok** [technicalbirdVayuz/volleyball-unite-usa-issue-tracker#63]
Can we make all Canadian IP addresses redirect to the .ca and all the USA IP addresses direct to the .com site?

**Comment by technicalbirdVayuz** [comment]
You had a discussion about showing the flag at the bottom, Can we implement the same?

@rgnarok Please confirm

**Comment by rgnarok** [comment]
Yes we can do that too in that case what we need to do is create a page which will open in case a user will hit Volleyballunite.com and see 2 flags one of USA and other of Canada in case the user select Canada then it will be redirected to volleyballunite.ca
and in case USA it will continue to volleyballunite.com

**Comment by technicalbirdVayuz** [comment]
Please convey the same to Priya, This would save time

**Comment by vayuzpriya** [comment]
@rgnarok  Should I work on it or not ?

**Comment by vayuzpriya** [comment]
@rgnarok  No Update Here ???
@technicalbirdVayuz

**Comment by technicalbirdVayuz** [comment]
Please check #128

**Comment by vayuzpriya** [comment]
UI  Updated 
functional part needs to be done @technicalbirdVayuz
